test/Array/array_qsortr_random.js

```diff
@@ -44,7 +44,7 @@ function stressTestSort(iterations, size = 1000)
     }
 }
 
-// Test 1000 random arrays of 1000 elements, print out the failures.
-stressTestSort(1000, 1000);
+// Test 1000 random arrays of 3000 elements, print out the failures.
+stressTestSort(1000, 3000);
```

Comment:
I'd like to see this test expanded to cover all of the cases where our behavior could vary: longer and shorter than the insertion/merge cutoff point, numbers and objects and strings, with and without comparators. For the object cases, part of the verification could be checking that the output was stable. We might have to knock down the iteration count to 100 or so to keep the test quick enough, but I think this could provide a good level of validation.